compiler/onecc-docker/README.md

```diff
@@ -0,0 +1,263 @@
+# onecc-docker
+
+`onecc-docker` supports `one-cmd`, which is currently only supported by ubuntu 18.04, in more OS.
+
+
+
+## Description
+
+For now, `one-cmds` tools only support Ubuntu 18.04 and 20.04(not officially).
+So people in other environments can't use our tools unless they upgrade the OS (or install Ubuntu OS).
```

Comment:
> they upgrade the OS

Actually, users may use recent Ubutuntu 22.04. Thus it is likely `they downgrade the OS`

Reply:
> Actually, users may use recent Ubutuntu 22.04. Thus it is likely they downgrade the OS

How about changing it like this?

Therefore, it is difficult for people in different environments to use our tools without using ubuntu18:04.